crates/vm/levm/src/gas_cost.rs

```diff
@@ -158,7 +158,12 @@ pub const CALLDATA_COST_NON_ZERO_BYTE: U256 = U256([16, 0, 0, 0]);
 // Blob gas costs
 pub const BLOB_GAS_PER_BLOB: U256 = U256([131072, 0, 0, 0]);
 
-pub fn exp(exponent_bits: u64) -> Result<U256, OutOfGasError> {
+pub fn exp(exponent: U256) -> Result<U256, OutOfGasError> {
```

Comment:
I think we can do even better in this function. If we make `exponent_bits` an `U256` then we can avoid the `ConversionError` (and remove its declaration in `errors.rs`), then operate with `U256`s to finally get `exponent_byte_size` as `U256`, avoiding the `.into()` done afterwords (as it is already an `U256`).

Reply:
I changed the type to `U256` and then I used the `checked_div` in `exponent_byte_size`.

Reply:
Great! We'll keep it this way for now, but take into account that divisions with constant non-zero divisors do not strictly need to be checked.